3.10/aql/functions-miscellaneous.md

```diff
@@ -446,6 +446,74 @@ guaranteed to remain the same in future versions of ArangoDB. The hash values
 should therefore be used only for temporary calculations, e.g. to compare if two
 documents are the same, or for grouping values in queries.
 
+### MINHASH()
+
+`MINHASH(values, numHashes) → hashes`
```

Comment:
Would be nice to provide an example of how it can be used together with stored values.

Reply:
Good suggestion, but that should not be in the AQL reference. The idea is to get the 3.10 reference docs done first, and then we can add additional content like guides (for MinHash, I would like to add an example page under ArangoSearch).